src/Uno.UITest.Helpers/AppInitializer.cs

```diff
@@ -50,7 +50,7 @@ public class AppInitializer
 		/// </summary>
 		/// <remarks>This method is generally called from the type constructor of a test assembly.</remarks>
 		/// <returns>An <see cref="IApp"/> instance representing the running application.</returns>
-		public static IApp ColdStartApp(Xamarin.UITest.Configuration.AppDataMode? mode = null)
+		private static IApp ColdStartAppCore(Xamarin.UITest.Configuration.AppDataMode? mode = null)
```

Comment:
I still don't understand why you need to place `AppDataMode` nullable. Can't you use the current default as the for the parameter-less version of AttachToApp ?

Reply:
@jeromelaban There are inconsistencies for the modes used, so I had to use nullable mode to indicate that and keep the old behavior as is. For example, `CreateiOSApp` and `CreateAndroidApp`. One of them was using `Auto` and the other was using `DoNotClear`. If I made the mode non-nullable, how would I keep this old behavior?

Reply:
Let's keep do not clear for both as this was the observed behavior.

Reply:
@jeromelaban But for cloud platforms, the default was "Clear".

Reply:
It's not been used extensively in this case. Let's change the default for that case and flag it as a breaking change.